docker-compose.yml

```diff
@@ -16,25 +16,54 @@ services:
       retries: 5
 
   plasma-deployer:
-    image: omisego/plasma-deployer:32e8058
-    environment:
-      - ETH_CLIENT_HOST=geth
+    build:
+      context: .
+      dockerfile: plasma-contract.Dockerfile
+    command:
+      - /bin/sh
+      - -c
+      - |
+          apk add --update curl
+          cd /home/node/plasma-contracts/plasma_framework
+          npx truffle migrate --network infura
+          cd build
+          echo '{"contracts":' > db.json && cat outputs.json >> db.json && echo '}' >> db.json
+          npx json-server -w db.json -p 8000 -H 0.0.0.0
     ports:
       - "8000:8000"
     expose:
       - "8000"
-    restart: always
-    healthcheck:
-      test: curl plasma-deployer:8000
-      interval: 30s
-      timeout: 1s
-      retries: 5
+    environment:
+      # DEPLOYER_PRIVATEKEY is the geth dev account initially funded address
+      - DEPLOYER_PRIVATEKEY=d885a307e35738f773d8c9c63c7a3f3977819274638d04aaf934a1e1158513ce
+      - MAINTAINER_PRIVATEKEY=6f30f140fd4724519e5017c0895f158d68bbbe4a81c0c10dbb25a0006e348807
+      - AUTHORITY_PRIVATEKEY=7f30f140fd4724519e5017c0895f158d68bbbe4a81c0c10dbb25a0006e348807
+      - INFURA_URL=http://geth:8545
+      - INFURA_API_KEY=
+      - MIN_EXIT_PERIOD=20
```

Comment:
This MIN_EXIT_PERIOD doesn't work, I think because plasma-deployer appears to take the value from EXIT_PERIOD_SECONDS instead 

https://github.com/omisego/plasma-deployer/blob/32e80582cfdce179ddd49b0344fb6c43ff896f5c/lib/plasma_deployer/deploy.ex#L23

Reply:
actually @kevsul this doesn't use plasma-deployer anymore! this goes straight into contracts!